src/WebJobs.Script/ScriptHostBuilderExtensions.cs

```diff
@@ -407,7 +408,34 @@ public static IHostBuilder SetAzureFunctionsConfigurationRoot(this IHostBuilder
             return builder;
         }
 
-        internal static void ConfigureApplicationInsights(HostBuilderContext context, ILoggingBuilder builder)
+        internal static void ConfigureTelemetry(this ILoggingBuilder loggingBuilder, HostBuilderContext context)
+        {
+            TelemetryMode mode;
+            var telemetryModeSection = context.Configuration.GetSection(ConfigurationPath.Combine(ConfigurationSectionNames.JobHost, ConfigurationSectionNames.TelemetryMode));
```

Comment:
This looks like the same logic as the other place you retrieve `TelemetryMode` from the config - should you extract to a common helper?

Reply:
I will have a look at this later.